uint8 marker_row_id
uint8 marker_col_id
float64 deltaX
float64 deltaY
float64 heading
